client/src/API/Questions.js

```diff
@@ -1,5 +1,119 @@
-const Questions = {
+const questionAPI = {
+  getAllQuestions: (product_id) => {
+    var options = { method: 'GET' };
```

Comment:
I think the method: GET is already handled in the index.js server file, but not sure it matters